<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>BatchLogImporterWidget</class>
 <widget class="QWidget" name="BatchLogImporterWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>511</width>
    <height>560</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Batch log import</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../../programs/UrgBenri/UrgBenri.qrc">
    <normaloff>:/icons/app_icon</normaloff>:/icons/app_icon</iconset>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QGroupBox" name="inputGroup">
     <property name="title">
      <string>CSV file (*.csv) folder</string>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="BusyIndicatorWidget" name="loadingIncator" native="true"/>
      </item>
      <item>
       <widget class="QLineEdit" name="inputFilenameLine">
        <property name="readOnly">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="loadInputButton">
        <property name="maximumSize">
         <size>
          <width>32</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset resource="../../programs/UrgBenri/UrgBenri.qrc">
          <normaloff>:/icons/load_icon</normaloff>:/icons/load_icon</iconset>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="stepsGroup">
     <property name="title">
      <string>Steps</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_23">
        <item>
         <widget class="QLabel" name="label_2">
          <property name="text">
           <string>Front:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSpinBox" name="frontStep">
          <property name="maximum">
           <number>5000</number>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_26">
        <item>
         <widget class="QLabel" name="label_5">
          <property name="text">
           <string>Total:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSpinBox" name="totalStep">
          <property name="maximum">
           <number>5000</number>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="stepsGroup_2">
     <property name="title">
      <string>Distance</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_22">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_24">
        <item>
         <widget class="QLabel" name="label_3">
          <property name="text">
           <string>Min:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSpinBox" name="minDistance">
          <property name="maximum">
           <number>5000</number>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_25">
        <item>
         <widget class="QLabel" name="label_4">
          <property name="text">
           <string>Max:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSpinBox" name="maxDistance">
          <property name="maximum">
           <number>5000</number>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <widget class="QProgressBar" name="exportProgress">
     <property name="value">
      <number>0</number>
     </property>
    </widget>
   </item>
   <item>
    <widget class="Line" name="line">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="importButton">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>Import</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="closeButton">
       <property name="text">
        <string>Close</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>BusyIndicatorWidget</class>
   <extends>QWidget</extends>
   <header>BusyIndicatorWidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="../../programs/UrgBenri/UrgBenri.qrc"/>
 </resources>
 <connections/>
</ui>
